addons/web_editor/static/src/scss/wysiwyg.scss

```
@@ -343,12 +343,20 @@ $o-we-zindex: $o-we-overlay-zindex + 1 !default;
         }
 
         .o_we_colorpicker_switch_panel button {
+            --bg: transparent;
+
             margin-bottom: -1px;
+            border: 0;
+            background: var(--bg);
```

I'm not sure `--bg` is the appropriate variable to use, since it handles the background of `.colorpicker .o_colorpicker_sections`.
Also, color picker in website seems to use directly the `transparent` value (https://github.com/odoo/odoo/blob/17.0/addons/web_editor/static/src/scss/wysiwyg_snippets.scss#L163)

I think you can set directly the `transparent` value.

```suggestion
            background: transparent;
```